examples/pvc-dp-cache.yaml

```diff
@@ -0,0 +1,114 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dynamic-plugins-root
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi
```

Comment:
IIUC, given the volume claim templates in the CR, I guess this PVC is useless in this example and won't be bound to any pod, no?

Reply:
+1